bugbug/models/qaneeded.py

```diff
@@ -64,7 +67,7 @@ def get_labels(self):
 
             for entry in bug_data['history']:
                 for change in entry['changes']:
-                    if change['added'].startswith('qawanted'):
+                    if change['added'].startswith('qawanted') or change['added'].startswith('qe-verify'):
                         classes[bug_id] = 1
                     elif 'flags' in entry:
```

Comment:
The second part here might have the be removed if it is unnecessary. Can you check if there is any bug where this happens?

Reply:
Yes, there are quite a few bugs with added as qe-verify.

Reply:
I'm not talking about the part you added (we talked about that already, I suggested you to add it :P).
I'm talking about the second part of the pre-existing code, which is analyzing flags.

Reply:
Oh! I just checked: there are around 1235 bugs with `name` as `qe-verify`. I think we should let this be here. Right?

Reply:
Can you show me an example? I want to check the difference between one of them and 1389223.

Reply:
(Also, if there are cases like that, then you need to add support for it to rollback)

Reply:
> Can you show me an example? I want to check the difference between one of them and 1389223.

Sure, for example bug id 1389252.

Reply:
1389252 doesn't match that: https://bugzilla.mozilla.org/rest/bug/1389252/history.

Reply:
Oh, sorry. I was looking at the wrong `flags` key.